lib/core/utils/valid-langs.js

```diff
@@ -1,8153 +1,105 @@
+
+/**
+ * Determine if a string is a valid language code
+ * @method isValidLang
+ * @memberof axe.utils
+ * @param {String} lang String to test if a valid language code
+ * @returns {Boolean}
+ */
+function isValidLang(lang) {
+	let array = langs;
+
+	// padEnd is not supported in IE11
+	while (lang.length < 3) {
+		lang += '`';
+	}
+	for (let i = 0; i <= lang.length - 1; i++) {
+		const index = lang.charCodeAt(i) - 96;
```

Comment:
Yesterday morning I didn't know about charCodeAt. Last night when working on a personal project, I actually needed it to deal with emoji chars. :) Funny how these things can go.